bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonNetworkHandler.java

```diff
@@ -0,0 +1,148 @@
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.insteon.internal.handler;
+
+import java.util.List;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.io.transport.serial.SerialPortManager;
+import org.openhab.binding.insteon.internal.InsteonBinding;
+import org.openhab.binding.insteon.internal.config.InsteonNetworkConfiguration;
+import org.openhab.binding.insteon.internal.discovery.InsteonDeviceDiscoveryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link InsteonNetworkHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Rob Nielsen - Initial contribution
+ */
+@NonNullByDefault
+@SuppressWarnings("null")
+public class InsteonNetworkHandler extends BaseBridgeHandler {
+
+    private static final int LOG_DEVICE_STATISTICS_DELAY_IN_SECONDS = 600;
+    private static final int SETTLE_TIME_IN_SECONDS = 5;
+
+    private final Logger logger = LoggerFactory.getLogger(InsteonNetworkHandler.class);
+
+    private @Nullable InsteonNetworkConfiguration config;
+    private @Nullable InsteonBinding insteonBinding;
+    private @Nullable InsteonDeviceDiscoveryService insteonDeviceDiscoveryService;
+    private @Nullable ScheduledFuture<?> pollingJob = null;
+    private @Nullable ScheduledFuture<?> settleJob = null;
+    private long lastInsteonDeviceCreatedTimestamp = 0;
+    private @Nullable SerialPortManager serialPortManager;
+
+    public static ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+
+    public InsteonNetworkHandler(Bridge bridge, @Nullable SerialPortManager serialPortManager) {
+        super(bridge);
+        this.serialPortManager = serialPortManager;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        logger.info("Starting Insteon bridge");
+        config = getConfigAs(InsteonNetworkConfiguration.class);
+
+        scheduler.execute(() -> {
+            insteonBinding = new InsteonBinding(this, config, serialPortManager);
+
+            updateStatus(ThingStatus.ONLINE);
```

Comment:
No yet, but this is needed due to the way the original binding was written. Changing it would be a major refactoring that I was trying to avoid doing with the port of the binding.

Reply:
The above warn message was changed as suggested.

Reply:
I guess setting it to ONLINE worked, I changed it as suggested.

Reply:
Setting to UNKNOWN didn't break anything. implemented as suggested.